apps/web/src/app/page.tsx

```diff
@@ -1,17 +1,42 @@
 import { getServerClient } from "@/utils/trpc/serverClient"
 import { getServerSession } from "next-auth"
+import { Button } from "../../../../packages/ui"
+import CompanySplash from "@/components/molecules/CompanySplash/CompanySplash"
+import { ComingEvent } from "@/components/molecules/ComingEvent/ComingEvent"
+import Link from "next/link"
 
 export default async function App() {
   const serverClient = await getServerClient()
-  const events = await serverClient.event.all()
-  const auth = await getServerSession()
+  const session = await getServerSession()
+  const events = await serverClient.event.recommended()
 
   return (
     <div>
-      <h1>Auth:</h1>
-      <pre>{JSON.stringify(auth, null, 4)}</pre>
-      <h1>Events:</h1>
-      <pre>{JSON.stringify(events, null, 4)}</pre>
+      <CompanySplash className="hidden md:flex" />
```

Comment:
Kan vi få inn at det er egen side for innlogget vs utlogget allerede nå?

Reply:
Det kan vi se på ja